Design Patterns/Examples/CSharpDesignPatterns/CSharpDesignPatterns.Singleton/Models/Animal.cs

```diff
@@ -0,0 +1,36 @@
+namespace CSharpDesignPatterns.Singleton.Models;
+
+internal class Animal
```

Comment:
Might also be worth showing a thread-safe Singleton? 
e.g 
https://github.com/MathewBeldon/DesignPatterns.Example/blob/main/Creational/Singleton/TheSingleton.cs

Reply:
I'll ask Ste if that type of stuff will have been covered or not. They shouldn't be seeing anything new in here in terms of language/framework features.